.github/workflows/validate_csharp_quicstarts.yaml

```diff
@@ -29,39 +29,29 @@ on:
       - release-*
 jobs:
   validate:
-    name: Validate quickstart for `${{ matrix.quickstart_language }}` with `${{ matrix.quickstart_variant }}` on ${{ matrix.os }}
+    name: Validate C# quickstart on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 15
+    timeout-minutes: 130
```

Comment:
With Earlier workflow, all the jobs used to run independently in parallel. With this change, all the building blocks for each language will run sequentially, due to which it will take more time complete the job.